code/Language/Drasil/ExprExtract.hs

```diff
@@ -0,0 +1,50 @@
+module Language.Drasil.ExprExtract (egetDoc)where
+
+import Control.Lens ((^.))
+import Language.Drasil.Document
+import Language.Drasil.Chunk.Eq (QDefinition)
+import Language.Drasil.Unit(UnitDefn)
+import Language.Drasil.Chunk.Quantity
```

Comment:
Please make sure that your `import` statements list all you import -- a lot of work has gone into other files to make sure this is the case.

Reply:
Import has been improved and all the warning were removed in 3ecbfc6469af72ea800ba7eddd1a510f5aee2e67.